codegen-client/src/test/kotlin/software/amazon/smithy/rust/codegen/client/smithy/endpoint/EndpointsDecoratorTest.kt

```diff
@@ -184,6 +188,12 @@ class EndpointsDecoratorTest {
                                             .a_bool_param(false)
                                             .a_string_param("hello".to_string())
                                             .region("us-east-2".to_string())
+                                            .a_string_array_param(
+                                                vec!["a", "b", "c"]
+                                                    .iter()
+                                                    .map(|i| i.to_string())
+                                                    .collect::<Vec<String>>()
```

Comment:
you may even be able to do the slightly more idiomatic:
```suggestion
                                                    .map(ToString::to_string)
                                                    .collect::<Vec<_>>()
```